src/main/java/Event.java

```diff
@@ -0,0 +1,39 @@
+public class Event extends Deadline{
+    private static final String TASKTITLE ="E";
+
+    public Event(int size){
+        super(size);
+    }
+
+    @Override
+    public String getBy(String line){
+        String[] lineWords= line.split("/",2);
+        String by;
+        try{
+            by = lineWords[1];
+
+        }catch(ArrayIndexOutOfBoundsException e) {
+            by ="No Event time!! Noice!!! :)";
```

Comment:
Inconsistent spacing between parentheses ( and { , as well as for the comma.

Reply:
Noted!